### PR TITLE
CP-908 Add "development" section to readme.

### PR DESCRIPTION
**Pull request by evanweible-wf**
## Issue
#35 

## Changes
Add dev section to readme. Points to dart_dev.

## Areas of Regression
n/a

## Testing
n/a

## Code Review
@trentgrover-wf
@maxwellpeterson-wf 
@dustinlessard-wf
@jayudey-wf

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -48,4 +48,12 @@ fluri
...
 
 fluri.updateQuery({'bar': '10'});
-```
\ No newline at end of file
+```
+
+
+## Development
+
+Truss leverages [the dart_dev package](https://github.com/Workiva/dart_dev) for
```
Comment (trentgrover-wf) [inline README.md]: truss?

Comment (trentgrover-wf) [inline README.md]: For copy/paste friendliness, could just get in the habit of saying `This project leverages...`

Comment (evanweible-wf) [inline README.md]: good idea, fixed

**Comment by trentgrover-wf** [comment]
+1

**Comment by trentgrover-wf** [comment]
@jayudey-wf docs only change

**Comment by jayudey-wf** [comment]
QA +1, doc change only

Merging into master

**Comment by jayudey-wf** [comment]
@rosie run_merge_script

**Pull request merged**